Project1/MobilityList.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <array>
```

Comment:
Don't need this; its in EnumContainer.

Reply:
No it isn't, but I will move it to there.

Reply:
No, if EnumContainer doesn't use it, it shouldn't have it. I thought it did though.